### PR TITLE
rb: add new w3c get cookie and delete all cookies functionality

### PR DESCRIPTION
**Pull request by lmtierney**
- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: rb/spec/integration/selenium/webdriver/options_spec.rb [hunk 1]
```diff
@@ -77,27 +77,35 @@ module WebDriver
...
             not_compliant_on browser: :edge do
-              it 'should delete all' do
+              it 'should delete one' do
                 driver.navigate.to url_for('xhtmlTest.html')
-
                 driver.manage.add_cookie name: 'foo', value: 'bar'
-                driver.manage.delete_all_cookies
-                expect(driver.manage.all_cookies).to be_empty
+
+                driver.manage.delete_cookie('foo')
```
Comment (twalpole) [inline rb/spec/integration/selenium/webdriver/options_spec.rb]: This test doesn't verify the cookie is deleted

Comment (lmtierney) [inline rb/spec/integration/selenium/webdriver/options_spec.rb]: apparently the spec never verified it was deleted... I'll add it in, thanks!

Comment (lmtierney) [inline rb/spec/integration/selenium/webdriver/options_spec.rb]: this has been added

**Comment by titusfortner** [comment]
merged: 68d51cf4494829c464a21e79fe0fd23bb752e7d9

**Pull request closed without merging**